src/common/components/FallbackComponent/index.jsx

```diff
@@ -1,20 +1,25 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { Button, View, Linking } from "react-native";
 import RNRestart from "react-native-restart";
 import { SafeAreaProvider, SafeAreaView } from "react-native-safe-area-context";
 import useThemedStyles from "@common/hooks/useThemedStyles";
 import { CustomText } from "@common";
 import constant from "../../constant";
+import { trackScreenView } from "../../firebase/analytics";
 import { logMessage } from "../../firebase/crashlytics";
-import useScreenAnalytics from "../../hooks/useScreenAnalytics";
 import STRINGS from "../../localization";
 import createStyles from "./styles";
 
 const FallBack = () => {
   const styles = useThemedStyles(createStyles);
-  logMessage(constant.FALLBACK);
   const { container, title, text, btnWrap } = styles;
-  useScreenAnalytics(constant.FALLBACK);
+
+  useEffect(() => {
+    // Track screen view when error fallback is shown
+    trackScreenView("ErrorFallback", null, "Error Boundary Fallback Screen");
```

Comment:
Why aren't we using the constant variable `constant.FALLBACK` here?

Reply:
fixed.